src/ui.button-toggle-item/UToggleItem.vue

```diff
@@ -30,12 +30,12 @@ const emit = defineEmits([
 ]);
 
 /* eslint-disable prettier/prettier */
-const getToggleName = inject("getToggleName", () => "toggle");
+const getToggleName = inject<() => string>("getToggleName", () => "toggle");
 const getToggleType = inject<() => string>("getToggleType", () =>
   getDefault<ToggleInjectValues>(defaultConfig, UToggleItem).type || TYPE_RADIO
 );
-const getToggleSize = inject<() => string>("getToggleSize", () =>
-  getDefault<ToggleInjectValues>(defaultConfig, UToggleItem).size || "md"
+const getToggleSize = inject<() => ButtonSize>("getToggleSize", () =>
+  getDefault<ToggleInjectValues>(defaultConfig, UToggleItem).size as ButtonSize || "md" as ButtonSize
```

Comment:
` (getDefault<ToggleInjectValues>(defaultConfig, UToggleItem).size || "md") as ButtonSize`

Should work too.